### PR TITLE
Update doc model_dann.py

### PR DESCRIPTION
**Pull request by lisab00**

### CODE REVIEW

File: domainlab/models/model_dann.py [hunk 1]
```diff
@@ -8,21 +8,45 @@
...
         type. Defaults to AModelClassif.
 
     Returns:
         ModelDAN: model inheriting from parent class
-    """
+
+    Input Parameters:
+        list_str_y: list of labels,
+        list_str_d: list of domains,
+        alpha: total_loss = task_loss + $$\\alpha$$ * regularization_loss,
```
Comment (smilesun) [inline domainlab/models/model_dann.py:34]: maybe put a hint here regularization loss here is the domain classification loss?

**Review by smilesun — CHANGES_REQUESTED** [review]
